forge.config.ts

```diff
@@ -57,6 +58,9 @@ const config: ForgeConfig = {
   },
   rebuildConfig: {},
   makers: [
+    new MakerPortable({
+			...winSigningConfig,
```

Comment:
This produce a portable .exe but with a generic icon, didn't find yet how to properly set the icon :(
Still digging.